lib/spanner/config/semantic_validator.ex

```diff
@@ -13,10 +13,10 @@ defmodule Spanner.Config.SemanticValidator do
   """
   @spec validate(Map.t) :: :ok | {:error, [{String.t, String.t}]}
   def validate(config) do
-    rules       = Map.fetch!(config, "rules")
-    bundle      = Map.fetch!(config, "bundle")
-    commands    = Map.fetch!(config, "commands")
-    permissions = Map.fetch!(config, "permissions")
+    rules       = Map.get(config, "rules", [])
+    bundle      = Map.get(config, "bundle")
+    commands    = Map.get(config, "commands")
+    permissions = Map.get(config, "permissions", [])
```

Comment:
I think we still want to call `fetch!` for "bundle" and "commands" since the code will blow up later on if they're `nil`.